github-dark.css

```diff
@@ -4810,7 +4810,7 @@
   .delete-topic-button {
     color: inherit !important;
   }
-  svg:not(.octicon):not([fill="none"]) {
+  svg:not(.octicon):not([fill="none"]), .markdown-body .octicon {
     fill: currentColor !important;
```

Comment:
That takes care of all the octicons on that page just fine =)

Reply:
I'll modify this line with the selector @Mottie suggested.